File: Turtle_puzzle.c
#include <stdio.h>
#include <stdlib.h>


int main(){

    int testes, tamanho, soma, i, numero;

    scanf("%d", &testes);

    while (testes > 0)
    {
        soma = 0;
        scanf("%d", &tamanho);

        for(i = 0; i < tamanho; i++){
            
            scanf("%d", &numero);

            if (numero <0){
                numero = abs(numero);
            }

            soma+= numero;
        }

        printf("%d\n", soma);
        testes--;
    }
    
    
    return 0;
}

File: Turtle_puzzle.py
testes = int(input())


while testes > 0 :

    tamanho = int(input())
    soma = 0

    for i in range(0, tamanho, 1):

        numero  = int(input())

        if numero < 0:
            numero = abs(numero)
        
        soma += numero

    print(soma)
    testes -= 1

